### DISCUSSION TITLE
Ability to skip database url input while scaffolding a new project

### DISCUSSION THREAD

**Discussion by anasmohammed361** [payloadcms/payload#8968] [Feature Requests & Ideas]
I have been playing around with payload lately , and i use docker for my local environment setup (databases and other necessary services for local development ) , everysingle time , i am forced to find an alternate solution when creating a new payload app , because payload requires an active database url which runs its migrations if necessary , It would be very convenient if there was a way to skip this process on project scaffolding .

**Comment by DanRibbens** [comment]
When you create a new Payload app with `create-payload-app`, it doesn't connect. I'm guessing you're talking about during development. There is a config setting for that which you could set an env variable for. 

Take a look at [disableConnect](https://payloadcms.com/docs/getting-started/installation#disabledbconnect).

Does that solve this issue for you?

**Reply by revnelson** [reply]
How can we do this in v3 beta?
